### PR TITLE
1283 comp list of script references

### PR DESCRIPTION
**Pull request by IlseFerris**
Updated multiple script's policies and/or procedures (even more than I had originally planned) from when we removed this information from SPO, and moved it to the CLoS. All tested and committed individually.

### CODE REVIEW

File: COMPLETE LIST OF SCRIPTS.vbs [hunk 1]
```diff
@@ -1774,7 +1774,7 @@ script_array(script_num).release_date           = #10/01/2000#
 script_array(script_num).hot_topic_date			= #08/31/2021#
 script_array(script_num).hot_topic_link			= "https://hennepin.sharepoint.com/teams/hs-economic-supports-hub/SitePages/New-Scripts-Available-September-1-for-Interview-and-Expedited-SNAP.aspx"
 script_array(script_num).used_for_elig			= False
-script_array(script_num).policy_references		= array("")						'SEE Line 58 for format'
+script_array(script_num).policy_references		= array("CM Emergency_Aid_Eligibility_-_SNAP/Expedited_Food 04.04", "TE Expedited_Food_Support_Tip_Sheet 13.031", "TE Expedited_SNAP_W_Pending_Verif's 02.10.01", "TE WREG_Expedited_SNAP_Postponed_Verifs 02.05.70.01", "TE QTIP_#152_-_Expedited_Food_Support 19.152", "TE EBT_-_Expedited_Food_Support 16.09", "TE Expedited_FS_2nd_Month_Eligibility TE02.10.79", "TE Asset_Coding_for_Food_Support 02.08.185", "SHAREPOINT Acceptable_Verifications https://hennepin.sharepoint.com/teams/hs-es-manual/SitePages/Acceptable_Verification.aspx")
```
Comment (MeganGeissler) [inline COMPLETE LIST OF SCRIPTS.vbs:1777]: I think you have an extra "TE" at the end of this portion:
"TE Expedited_FS_2nd_Month_Eligibility TE02.10.79"

File: COMPLETE LIST OF SCRIPTS.vbs [hunk 2]
```diff
@@ -4598,7 +4598,7 @@ script_array(script_num).subcategory            = array("")
 script_array(script_num).release_date           = #10/01/2000#
 script_array(script_num).hot_topic_link			= ""
 script_array(script_num).used_for_elig			= False
-script_array(script_num).policy_references		= array("")						'SEE Line 58 for format'
+script_array(script_num).policy_references      = array("CM Emergency_Aid_Eligibility_-_SNAP/Expedited_Food 04.04", "TE Expedited_Food_Support_Tip_Sheet 13.031", "TE Expedited_SNAP_W_Pending_Verif's 02.10..01", "TE WREG_Expedited_SNAP_Postponed_Verifs 02.05.70.01")
```
Comment (MeganGeissler) [inline COMPLETE LIST OF SCRIPTS.vbs:4601]: Extra "." between 10 and 01. 
"TE Expedited_SNAP_W_Pending_Verif's 02.10..01"

File: COMPLETE LIST OF SCRIPTS.vbs [hunk 3]
```diff
@@ -717,7 +717,7 @@ script_array(script_num).subcategory            = array("")  '<<Temporarily remo
 script_array(script_num).release_date           = #10/01/2000#
 script_array(script_num).hot_topic_link			= ""
 script_array(script_num).used_for_elig			= False
-script_array(script_num).policy_references		= array("")						'SEE Line 58 for format'
+script_array(script_num).policy_references		= array("CM Emergency_Aid_Eligibility_-_SNAP/Expedited_Food 04.04","CM Applications 05","CM Application_-_Pending Cases 05.09.12","TE Notice_of_Interview/Missed_Interview_(NOMI) 02.05.15","TE Expedited_Food_Support_Tip_Sheet 13.031", "SHAREPOINT Applications https://hennepin.sharepoint.com/teams/hs-es-manual/SitePages/Applications.aspx")						'SEE Line 58 for format')						'SEE Line 58 for format'
```
Comment (MeganGeissler) [inline COMPLETE LIST OF SCRIPTS.vbs:720]: "CM Application_-_Pending Cases 05.09.12"give me an error in the browser: access to dhs.state.mn.us was denied. However I am able to pull up 05.09.12 on a separate tab with no issues. Not sure if this is a me issue or if the "-" in the name is possibly problematic?

File: COMPLETE LIST OF SCRIPTS.vbs [hunk 4]
```diff
@@ -1595,7 +1595,7 @@ script_array(script_num).subcategory            = array("")
 script_array(script_num).release_date           = #03/05/2019#
 script_array(script_num).hot_topic_link			= ""
 script_array(script_num).used_for_elig			= False
-script_array(script_num).policy_references		= array("")						'SEE Line 58 for format'
+script_array(script_num).policy_references		= array("CM Determining_Gross_Income 17", "TE Student_Income 02.08.087", "TE AmeriaCorps 02.05.34", "TE SNAP_Benefits_-_2nd_30_Days_Applicant_Delay 02.05.107", "TE CASE_NOTE_II_Assets_Income 02.08.094", "TE Determining MFIP_Initial/Ongoing_Eligibilty 02.05.80", "TE Budget_Cycle/Monthly_Reporting_Changes 02.080.32", "SHAREPOINT Income https://hennepin.sharepoint.com/teams/hs-es-manual/SitePages/Income.aspx", "SHAREPOINT Earned_Income https://hennepin.sharepoint.com/teams/hs-es-manual/SitePages/Earned_Income.aspx")						'SEE Line 58 for format'
```
Comment (MeganGeissler) [inline COMPLETE LIST OF SCRIPTS.vbs:1598]: Missing a "_" between Determining and MFIP causing the TE# not to show up in the spreadsheet.
"TE Determining MFIP_Initial/Ongoing_Eligibilty 02.05.80"

Comment (MeganGeissler) [reply COMPLETE LIST OF SCRIPTS.vbs:1598]: Extra "0" in the TE#, should be 02.80.32
"TE Budget_Cycle/Monthly_Reporting_Changes 02.080.32"

**Comment by MeganGeissler** [comment]
Added a few comments!

**Review by MDR22 — APPROVED** [review]
Found one minor typo besides Megan's comments. Otherwise, looks good!

File: COMPLETE LIST OF SCRIPTS.vbs (hunk 3 above)
Comment (C-Love) [reply COMPLETE LIST OF SCRIPTS.vbs:720]: This is not linking correctly, but it looks like it is missing an underscore between Pending and Cases. It should be "CM Application_-_Pending_Cases 05.09.12"

File: COMPLETE LIST OF SCRIPTS.vbs [hunk 5]
```diff
@@ -3916,7 +3916,7 @@ script_array(script_num).subcategory            = array("")
 script_array(script_num).release_date           = #1/31/2023#
 script_array(script_num).hot_topic_link			= ""
 script_array(script_num).used_for_elig			= False
-script_array(script_num).policy_references		= array("")						'SEE Line 58 for format'
+script_array(script_num).policy_references		= array("CM Application_-_Pending Cases 05.09.12")						'SEE Line 58 for format'
```
Comment (C-Love) [inline COMPLETE LIST OF SCRIPTS.vbs:3919]: There should be an underscore between 'Pending' and 'Cases'

**Review by C-Love — CHANGES_REQUESTED** [review]
There are some line specific comments with some links that didn't generate correctly. 

While checking these, I found one that has changed. 'REP-REVS List' has HC INCOME RENEWAL (CSR) was removed and replaced with Medical Assistance (MA) Renewals (https://hennepin.sharepoint.com/teams/hs-es-manual/SitePages/Medical-Assistance-(MA)-Renewals.aspx). I can make a new issue for this if that makes more sense.

There is also an issue on 'Significant Change' which is not on this issue, I can make a separate issue for this as well if needed.

I also created 2 associated issues #1464 and #1465, from reviewing all of this code. Especially because my eyes are now blurred from looking at all of these policy reference additions, ewww.

File: COMPLETE LIST OF SCRIPTS.vbs (hunk 4 above)
Comment (IlseFerris) [reply COMPLETE LIST OF SCRIPTS.vbs:1598]: Wow, you can really tell this was the last commit of the day.

**Pull request closed without merging**